bundles/net.tourbook/src/net/tourbook/database/TourDataUpdate_053_to_054.java

```diff
@@ -46,7 +47,8 @@ public boolean updateTourData(final TourData tourData) {
       // convert weather-showers-scatterd
       //    into weather-showers-scattered
 
-      if (tourData.getWeather_Clouds().equalsIgnoreCase("weather-showers-scatterd")) { //$NON-NLS-1$
+      if (StringUtils.hasContent(tourData.getWeather_Clouds()) &&
+            tourData.getWeather_Clouds().equalsIgnoreCase("weather-showers-scatterd")) { //$NON-NLS-1$
 
```

Comment:
Wouldn't it be a better solution?

![image](https://github.com/user-attachments/assets/42091d9f-df88-4f78-b7cf-508debec754e)

Reply:
Fixed